### DISCUSSION TITLE
Add object models in the frigate database

### DISCUSSION THREAD

**Discussion by bend94** [blakeblackshear/frigate#12709] [Ask A Question]
### What is your question:

Hi,
I am using frigate since a while now, very good product.
One question, in all the objects that frigate can recognize for instance car, bicycle, dog, etc
it is possible to train frigate and add a new object , for example the postman car ?

Thanks for your help
regards

**Comment by NickM-27** [comment]
Yes, custom models can be used https://docs.frigate.video/configuration/advanced#model
